src/SixLabors.Fonts/GlyphType.cs

```diff
@@ -0,0 +1,26 @@
+// Copyright (c) Six Labors and contributors.
+// Licensed under the Apache License, Version 2.0.
+
+namespace SixLabors.Fonts
+{
+    /// <summary>
+    /// Represents the various version of a glyph records.
+    /// </summary>
+    public enum GlyphType
+    {
+        /// <summary>
+        /// This is a fall back glyph dure to missing code point
```

Comment:
`due`?

Reply:
yup fat fingers all the way.. 😀 ..sorted.